articles/quickstart/webapp/rails/01-login.md

```diff
@@ -232,12 +232,15 @@ Use the following command to create the controller that will handle user logout:
 rails generate controller logout
 ```
 
+To clear out all the objects stored within the session, call the `reset_session` method within the `logout_controller/logout` method. Learn more about reset_session [here](http://api.rubyonrails.org/classes/ActionController/Base.html#M000668).
```

Comment:
> Learn more about reset_session [here](http://api.rubyonrails.org/classes/ActionController/Base.html#M000668).

Can you make this entire sentence a link, as it is on the second part of the tutorial? It gives the user much more context about what it is they're about to click on, especially if they're using a screen reader.